databases/week2/PhoneStore Database.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE phoneStore;
+
+USE phoneStore;
+
+CREATE TABLE phones (
+    `phone_id` INT PRIMARY KEY,
+    `brand` VARCHAR(50),
```

Comment:
Another point - please consider if you don't want any of those values to possibly be null (=empty). I recommend reconsidering the model here since any of those values here can be null -  it doesn't really make any sense to have a database if a user has a possibility of nullable inputs. To force a new input to have values, apply NOT NULL.

E.g.:

`'brand' VARCHAR(50) NOT NULL`